### PR TITLE
Allow usage of redis

### PR DESCRIPTION
**Pull request by sinneduy**

### CODE REVIEW

File: lib/gemstash/cli/setup.rb [hunk 1]
```diff
@@ -83,6 +84,13 @@ def ask_memcached_details
         @config[:memcached_servers] = servers
       end
 
+      def ask_redis_details
+        say_current_config(:redis_servers, "Current Memcached servers")
```
Comment (optix2000) [inline lib/gemstash/cli/setup.rb]: Should say Redis not Memcached

**Comment by zp-alfred** [comment]
Changes requested by optix2000.

**Comment by zp-alfred** [comment]
Changes requested by optix2000.

**Comment by zp-alfred** [comment]
Accepted by optix2000!

**Pull request merged**